runtime/caml/startup_aux.h

```diff
@@ -36,7 +36,6 @@ struct caml_params {
   uintnat verb_gc;
   uintnat parser_trace;
   uintnat trace_level;
-  uintnat runtime_events_enabled;
   uintnat runtime_events_log_wsize;
```

Comment:
Same as in the docs, can we rename that to `runtime_events_logwsize` to avoid confusion?